components/scream/src/physics/p3/tests/p3_unit_tests.cpp

```diff
@@ -46,12 +44,12 @@ struct UnitWrap::UnitTest<D>::TestP3Func
     for(int s = 0; s < sat_ice_p.n; ++s){
       // Test vapor pressure
       if (abs(sat_ice_p[s] - correct_sat_ice_p) > tol ) {errors++;}
-      if (abs(sat_liq_p[s] - correct_sat_liq_p) > tol)  {errors++;}
+      //if (abs(sat_liq_p[s] - correct_sat_liq_p) > tol)  {errors++;}
```

Comment:
What's the story here? Does this line need to be removed?

Reply:
Must have commented that one out while debugging. It should not be commented, comment removed and tests still pass.